website/src/pages/index.tsx

```diff
@@ -64,6 +85,11 @@ const Index: FC = () => (
         name="og:description"
         content="Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API gateway, based on the Nginx library and etcd."
       />
+
+      <script type="application/ld+json">
```

Comment:
Why don't you put the JSON directly here?

Reply:
It did not work when I directly added the JSON. I looked at an example implementation, and they used a similar approach.